src/lib/components/wallets/Wallets.svelte

```diff
@@ -15,12 +15,29 @@
     const dispatch = createEventDispatcher();
 
     let filteredWallets: IWallet[];
+    let orderedWallets: any;
     if (wallets.length) {
         filteredWallets = removeDuplicates(wallets).map(walletFactory.create);
     } else {
         filteredWallets = walletFactory.createAll();
     }
 
+    async function orderWallets(walletList: IWallet[]): Promise<IWallet[]> {
+        const orderedWalletList = [];
+        for (let wallet of walletList) {
+            const isInstalled = await wallet.isInstalled();
+
+            if (isInstalled) {
+                orderedWalletList.unshift(wallet);
+            } else {
+                orderedWalletList.push(wallet);
+            }
+        }
+        return orderedWalletList;
+    }
+
+    orderedWallets = orderWallets(filteredWallets);
```

Comment:
You can do `orderWallets().then((wallets) => (orderedWallets = wallets));` and avoid using await below in the HTML

Reply:
would doing `await orderWallets(...)` yield the same result?

Reply:
You can't use await in the script, only with an IIFE or an onMount